lib/smart_answer_flows/coronavirus-employee-risk-assessment/questions/is_your_workplace_an_exception.govspeak.erb

```diff
@@ -1,22 +1,47 @@
+<%
+  render_terse_options =  false
+  workplace =  calculator.where_do_you_work
+  if workplace == "museums_or_galleries" || workplace == "cinema" || workplace == "indoor_attraction"
+    render_terse_options = true
+  end
+%>
 <% render_content_for :title do %>
-  Is your workplace one of these exceptions?
+  <% if workplace == "museums_or_galleries" %>
+    Is your place of work a retail gallery?
+  <% elsif workplace == "cinema" %>
+    Is your place of work a drive-in cinema?
+  <% elsif workplace == "indoor_attraction" %>
+    Is your workplace an outdoor area of a zoo, safari park, wildlife centre or other animal attraction?
+  <% else %>
+    Is your workplace one of these exceptions?
+  <% end %>
```

Comment:
similarly here we could probably change all of this to:

```
<% case workplace %>
<% when "museums_or_galleries" %>
    Is your place of work a retail gallery?
  <% when "cinema" %>
    Is your place of work a drive-in cinema?
  <% when "indoor_attraction" %>
    Is your workplace an outdoor area of a zoo, safari park, wildlife centre or other animal attraction?
  <% else %>
    Is your workplace one of these exceptions?
  <% end %>
```

Reply:
Thanks for the tip @DilwoarH, your suggestion simplifies the code 👍